test/parallel/test-sync-trace-fs.js

```diff
@@ -0,0 +1,161 @@
+'use strict';
```

Comment:
To group this better with other trace event test cases, please use a file name like `test-trace-events-fs-sync.js`